quickstack/manifests/compute_common.pp

```diff
@@ -464,8 +465,10 @@
     repo_server => $repo_server,
     randomwait => 240,
   }
-  
-  class {'moc_openstack::suricata':
+
+  if !$sahara_enabled {   
+    class {'moc_openstack::suricata':
+    }
```

Comment:
If we can do our best to ensure that in whatever environments we deploy Sahara in the controller can always access instances over internal IP (10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16), then no further modifications to Suricata are necessary and we can keep it enabled as-is. If this is the case, then I should also set `use_floating_ips => false` in ::sahara.   

If we cannot rely on the accessibility of instances over internal IP then we must add a rule in moc_openstack/files/pass.rules to allow traffic between the controller and whatever public subnet may be in use. This is where I was not sure how to proceed since the public network could be 129.10.5.0/24, 129.10.3.0/24, or something entirely different depending on environment and I wasn't too keen on hard coding these values in.